### PR TITLE
fix: Argos tests

### PR DESCRIPTION
**Pull request by bprusinowski**
Update GitHub e2e action workflow to always run screenshot upload step.

### CODE REVIEW

**Comment by bprusinowski** [comment]
@ptbrowne it seems that I can't test this action until it's on `main` branch... I think there was a problem with the screenshot upload step, it didn't got to that point if a failure in e2e tests appeared before it. I've added an `if: always()` condition and changed the action to use the one suggested in the docs (https://argos-ci.com/docs/github-actions#complete-example)

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -59,14 +59,6 @@ jobs:
...
-        run: echo "ARGOS_BRANCH=$(git name-rev --name-only $GITHUB_SHA | sed 's-remotes/origin/--')" >> $GITHUB_ENV
-        if: github.ref != 'refs/heads/main'
-      - name: Upload screenshots to Argos
-        env:
-          ARGOS_COMMIT: ${{ github.sha }}
-          ARGOS_TOKEN: ${{ secrets.ARGOS_TOKEN }}
-        run: yarn run argos upload e2e-screenshots
+      - name: Upload screenshots to argos-ci.com
+        if: always()
+        run: npx @argos-ci/cli upload e2e-screenshots
```
Comment (ptbrowne) [inline .github/workflows/e2e.yml:64]: So now the argos CLI detects correctly the branch even in the case of PR branches ?

Comment (ptbrowne) [reply .github/workflows/e2e.yml:64]: I guess ? https://github.com/argos-ci/argos-javascript/blob/bad0f5b5bf1dc1672b05cf08a4fec5d011ac4b0a/packages/core/src/ci-environment/git.ts#LL19C31-L19C62

**Review by ptbrowne — APPROVED** [review]

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (bprusinowski) [reply .github/workflows/e2e.yml:64]: Yes, looks like!

**Pull request merged**